tbx/project_styleguide/templates/patterns/pages/work/work_index_page.html

```diff
@@ -0,0 +1,18 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block content %}
+    <div>
+        {% include "patterns/molecules/title-filters/title-filters.html" with item=page tags=related_services %}
+
+        {% for post in works %}
+            <ul>
+                {% include "patterns/molecules/listing/listing--image.html" with title=post.title client=post.client name=post.author.name job_title=post.author.role link=post.url date=post.date reading_time=post.read_time tags=post.related_services listing_image=post.listing_image tag_link_base="." tags_screenreader_text="Filter work posts by category" %}
```

Comment:
we no longer want the author in here as per the design changes that were discussed this morning, but I can sort this as part of the FE work.